azavea/python-omgeo
# Add HTTP/S support via Requests library

Replace use of `urllib` and `urllib2` with Requests library while attempting to maintain backwards compatibility.

Fixes #52 

---

**Testing**

- Ensure that the test suite passes (https://travis-ci.org/azavea/python-omgeo/builds/390959259)
- Ensure that a response object is still returned from `_get_response`
- Confirm all uses of `_get_response`, `_get_json_obj`, `_get_xml_doc` function
- Confirm `response.text` is the proper replacement for `response.read()`

(Added @KlaasH because this change impacts Cicero.)
